go.mod

```diff
@@ -3,14 +3,17 @@ module github.com/filecoin-project/go-amt-ipld/v2
 go 1.12
 
 require (
-	github.com/ipfs/go-block-format v0.0.2
+	github.com/filecoin-project/go-amt-ipld v1.0.0
```

Comment:
Importing itself?

Reply:
It is importing older version. It should not.